apps/ryal_core/lib/ryal/web/models/payment_gateways/stripe.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Ryal.PaymentGateway.Stripe do
+  @moduledoc "Relevant functions for working with the Stipe API."
+
+  alias Ryal.PaymentGatewayQuery
+
+  @stripe_api_key Map.get(Ryal.payment_gateways(), :stripe)
+  @stripe_base "https://#{@stripe_api_key}:@api.stripe.com"
+
+  @spec create(atom, Ecto.Schema.t, String.t) :: {:ok, String.t}
```

Comment:
I need to remove the need for an `Ecto.Schema.t` and use either multiple args or a map instead.